rca/project_styleguide/templates/patterns/atoms/sprites/sprites.html

```diff
@@ -120,4 +120,9 @@
4.684h-.525l-2.282-4.684v4.684H22.36ZM34.49 30.333c-.603 0-1.135-.13-1.595-.388a2.794 2.794 0 0 1-1.081-1.073c-.26-.456-.39-.977-.39-1.566 0-.643.127-1.201.383-1.678a2.804 2.804 0 0 1 1.054-1.11c.446-.262.961-.395 1.544-.395.617 0 1.146.147 1.58.436.432.29.753.698.96 1.226.209.527.28 1.147.22 1.858h-1.47V27.1c0-.6-.096-1.032-.287-1.294-.191-.262-.505-.395-.938-.395-.505 0-.876.153-1.115.46-.239.306-.358.759-.358 1.358 0 .548.12.974.358 1.273.239.3.59.45 1.05.45.29 0 .54-.065.747-.191a1.2 1.2 0 0 0 .474-.552l1.483.426a2.602 2.602 0 0 1-1.05 1.253c-.477.3-1.002.446-1.569.446Zm-1.954-2.693v-1.1h3.909v1.1h-3.909ZM38.21 30.17v-5.887h1.31v1.95h.185v3.936h-1.497.003Zm3.561 0v-3.477c0-.388-.092-.69-.276-.909-.184-.214-.437-.323-.76-.323-.325 0-.56.106-.744.32-.188.211-.28.487-.28.824l-.62-.436c0-.384.095-.732.29-1.035.194-.306.453-.548.777-.725a2.246 2.246 0 0 1 1.095-.265c.46 0 .842.098 1.139.292.296.195.515.457.658.783.143.327.211.688.211 1.08v3.867h-1.483l-.007.003Zm3.547 0v-3.477c0-.388-.092-.69-.276-.909-.185-.214-.437-.323-.761-.323a.995.995 0 0 0-.539.143.978.978 0 0 0-.358.402c-.085.17-.13.37-.13.596l-.62-.416c0-.401.095-.759.286-1.069.191-.31.447-.548.771-.721a2.223 2.223 0 0 1 1.07-.26c.673 0 1.184.198 1.525.597.345.398.515.919.515 1.562v3.867h-1.483v.007ZM48.21 30.17v-7.848h1.493v3.813h-.184v4.03h-1.31v.004Zm2.905.163c-.587 0-1.074-.136-1.47-.409a2.515 2.515 0 0 1-.887-1.11 4.049 4.049 0 0 1-.296-1.586c0-.589.099-1.117.293-1.587.198-.47.484-.837.866-1.113.382-.272.853-.408 1.415-.408.563 0 1.054.136 1.474.401.42.27.743.637.975 1.104.232.466.348 1 .348 1.6 0 .599-.113 1.116-.341 1.586-.229.47-.546.837-.952 1.11-.409.272-.883.408-1.425.408v.004Zm-.242-1.318c.32 0 .583-.082.784-.242.205-.16.355-.374.454-.646.099-.27.146-.572.146-.902 0-.33-.05-.627-.153-.896a1.428 1.428 0 0 0-.47-.647c-.212-.16-.485-.241-.816-.241-.31 0-.559.075-.75.224a1.25 1.25 0 0 0-.416.627c-.089.269-.13.582-.13.936 0 .354.045.667.13.936.089.27.228.477.43.627.197.15.463.224.794.224h-.003ZM57.771 30.333c-.603 0-1.135-.13-1.596-.388a2.795 2.795 0 0 1-1.08-1.073c-.26-.456-.39-.977-.39-1.566 0-.643.127-1.201.383-1.678a2.804 2.804 0 0 1 1.053-1.11c.447-.262.962-.395 1.545-.395.617 0 1.146.147 1.579.436.433.29.754.698.962 1.226.208.527.28 1.147.218 1.858h-1.47V27.1c0-.6-.095-1.032-.286-1.294-.191-.262-.505-.395-.938-.395-.505 0-.877.153-1.115.46-.239.306-.358.759-.358 1.358 0 .548.119.974.358 1.273.238.3.59.45 1.05.45.29 0 .539-.065.747-.191a1.2 1.2 0 0 0 .474-.552l1.483.426a2.602 2.602 0 0 1-1.05 1.253c-.478.3-1.003.446-1.569.446Zm-1.954-2.693v-1.1h3.908v1.1h-3.908ZM61.67 30.17v-5.887h1.309v1.44l-.143-.184c.075-.204.177-.388.307-.555a1.58 1.58 0 0 1 .47-.415c.143-.088.297-.156.464-.204.167-.048.341-.082.518-.092.178-.014.358-.007.536.014v1.385c-.164-.05-.355-.068-.57-.05a1.69 1.69 0 0 0-.586.149c-.174.078-.32.187-.444.32a1.34 1.34 0 0 0-.272.47c-.062.18-.093.381-.093.606v3.006H61.67v-.004Z"/></g><defs><clipPath id="a"><path fill="currentColor" d="M0 0h87v97H0z"/></clipPath></defs>
```

Comment:
Might be worth renaming the id to something more specific, to avoid duplicated id's on the page, `qaa-logo-clippath` perhaps?

Reply:
Nice one, thanks - updated.